src/librbd/image/CloneRequest.cc

```diff
@@ -124,6 +124,13 @@ void CloneRequest<I>::validate_options() {
     }
   }
 
+  static std::unordered_set<int> valid_clone_formats = {1, 2};
+  if (valid_clone_formats.find(m_clone_format) == valid_clone_formats.end()) {
+    lderr(m_cct) << "RBD_IMAGE_OPTION_CLONE_FORMAT should be equal to 1 or 2" << dendl;
```

Comment:
Using an STL container for two statically known integers is really overkill:
```suggestion
  if (m_clone_format < 1 || m_clone_format > 2) {
    lderr(m_cct) << "unsupported clone format: " << m_clone_format << dendl;
```